kit/src/components/message/mod.rs

```diff
@@ -256,7 +252,8 @@ fn ChatText(cx: Scope<ChatMessageProps>) -> Element {
         .map(|e| e.as_str().to_string())
         .collect();
 
-    let texts = finder.spans(&cx.props.text).map(|e| match e.kind() {
+    // this is broken. may be fixed later.
```

Comment:
does this need a ticket?

Reply:
Idk. I assume it formats links somehow but am not sure. Would have to verify with @Flemmli97